concepts/05 UI Components/FilterBuilder/010 Overview.md

```diff
@@ -72,6 +85,108 @@ The following code adds a simple FilterBuilder to your page. Note that each item
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxFilterBuilder
+            v-model:value="filterValue">
+            <DxField
+                data-field="Product_ID"
+                data-type="number"
+                caption="ID"
+            />
+            <DxField data-field="Product_Name" />
+            <DxField
+                data-field="Product_Cost"
+                data-type="number"
+                caption="Cost"
+                format="currency"
+            />
+        </DxFilterBuilder>
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.light.css';
+
+    import DxFilterBuilder, {
+        DxField
+    } from 'devextreme-vue/filter-builder';
+
+    export default {
+        components: {
+            DxFilterBuilder,
+            DxField
+        },
+        data() {
+            return {
+                filterValue: [
+                    [
+                        ["Product_Name", "startswith", "Super"],
+                        "and",
+                        ["Product_Cost", ">=", 300]
+                    ],
+                    "or",
+                    [
+                        ["Product_Name", "contains", "HD"],
+                        "and",
+                        ["Product_Cost", "<", 200]
+                    ]
+                ]
+            };
+        }
+    }
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import { useState } from 'react';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import FilterBuilder, {
+        Field
+    } from 'devextreme-react/filter-builder';
+
+    export default function App() {
+        const [filterValue, setFilterValue] = useState([
+            [
+                ["Product_Name", "startswith", "Super"],
+                "and",
+                ["Product_Cost", ">=", 300]
+            ],
+            "or",
+            [
+                ["Product_Name", "contains", "HD"],
+                "and",
+                ["Product_Cost", "<", 200]
+            ]
+        ]);
+
+        const handleFilterValueChange = ({ value }) => {
+            setFilterValue(value);
+        };
+
+        return (
+            <FilterBuilder
+                value={filterValue}
+                onValueChanged={handleFilterValueChange}>
```

Comment:
You can use onValueChange callback and remove handleFilterValueChange:
`onValueChange={setFilterValue}`